### PR TITLE
Tpm tasks

### PR DESCRIPTION
**Pull request by TimMcDowell**
Schema changes for the tasks assigned to me on the 1/4/2021 call

### CODE REVIEW

File: schema-source/RS0001.schema.yaml [hunk 1]
```diff
@@ -681,13 +681,13 @@ LookupVariablesCooling:
...
-    motor_controller_heat:
-      Description: "Heat rejected by the motor control center cooling"
+    auxiliary_heat:
+      Description: "Heat rejected by auxiliaries"
       Data Type: "[Numeric/Null][1..]"
       Units: "W"
       Range: ">=0.0"
       Required: True
-      Notes: "Heat transferred to another liquid crossing the control volume boundary from the chiller motor controller (inverter drive, starter, etc).\n\nSet as 0 if not present or if heat rejection is met by condenser"
+      Notes: "Heat transferred to another liquid crossing the control volume boundary from the chiller auxiliaries (motor controller, inverter drive, starter, etc).\n\nSet as 0 if not present or if heat rejection is met by condenser"
```
Comment (nealkruis) [inline schema-source/RS0001.schema.yaml:690]: Shouldn't we include the motor itself in the list of examples?

File: schema-source/RS0003.schema.yaml [hunk 2]
```diff
@@ -66,6 +66,9 @@ ComponentType:
     ELECTRIC_RESISTANCE_ELEMENT:
       Display Text: "Electric Resistance Element"
       Description: "Electric resistance heater elements"
+    DIRECT_EVAPORATIVE:
+      Display Text: "Direct Evaporative"
+      Description: "Wetted media cooling"
```
Comment (nealkruis) [inline schema-source/RS0003.schema.yaml:71]: "Wetted evaporative cooling media"?

**Review by nealkruis — CHANGES_REQUESTED** [review]
I added suggestions for minor changes. Feel free to accept/reject/modify.

File: schema-source/RS0001.schema.yaml (hunk 1 above)
Comment (TimMcDowell) [reply schema-source/RS0001.schema.yaml]: I added motor to the list

File: schema-source/RS0003.schema.yaml (hunk 2 above)
Comment (TimMcDowell) [reply schema-source/RS0003.schema.yaml]: I changed the wording - none of the wording sounds quite right though!

**Review by nealkruis — APPROVED** [review]

**Pull request merged**